{{cookiecutter.project_slug}}/compose/production/postgres/maintenance/restore

```diff
@@ -37,40 +37,18 @@ if [[ "${POSTGRES_USER}" == "postgres" ]]; then
     exit 1
 fi
 
+export PGHOST="postgres"
+export PGUSER="${POSTGRES_USER}"
 export PGPASSWORD="${POSTGRES_PASSWORD}"
-
-message_info "Dropping all connections to the database..."
-# Source: http://dba.stackexchange.com/a/11895
-drop_postgres_connections_sql='UPDATE pg_database'
-drop_postgres_connections_sql+=" SET datallowconn = 'false'"
-drop_postgres_connections_sql+=" WHERE datname = '${POSTGRES_DB}';"
-drop_postgres_connections_sql+='SELECT pg_terminate_backend(pid)'
-drop_postgres_connections_sql+=' FROM pg_stat_activity'
-drop_postgres_connections_sql+=" WHERE datname = '${POSTGRES_DB}';"
-psql \
-    --host=localhost \
-    --username=postgres \
-    --dbname=postgres \
-    --command="${drop_postgres_connections_sql}"
+export PGDATABASE="${POSTGRES_DB}"
 
 message_info "Dropping the database..."
-dropdb \
-    --host=postgres \
-    --username="${POSTGRES_USER}" \
-    "${POSTGRES_DB}"
+dropdb "${PGDATABASE}"
```

Comment:
This seems to fail when I try to follow the instructions, there are opened connections. I think I've followed the steps from the documentation:

```bash
> docker-compose -f local.yml up -d
Starting testrestore_postgres_1 ... done
Starting testrestore_django_1   ... done

> docker-compose -f local.yml run --rm postgres backup
Backing up the 'test_restore' database...
SUCCESS: 'test_restore' database backup 'backup_2018_05_01T13_25_26.sql.gz' has been created and placed in '/backups'.

> docker-compose -f local.yml run --rm postgres backups
These are the backups you have got:
total 12K
-rw-r--r-- 1 root root 5.2K May  1 13:25 backup_2018_05_01T13_25_26.sql.gz
-rw-r--r-- 1 root root   20 May  1 13:24 backup_2018_05_01T13_24_46.sql.gz

> docker-compose -f local.yml run --rm postgres restore backup_2018_05_01T13_25_26.sql.gz
Restoring the 'test_restore' database from the '/backups/backup_2018_05_01T13_25_26.sql.gz' backup...
INFO: Dropping the database...
dropdb: database removal failed: ERROR:  database "test_restore" is being accessed by other users
DETAIL:  There are 2 other sessions using the database.
```

Also the [example of output[(https://cookiecutter-django.readthedocs.io/en/latest/docker-postgres-backups.html#restoring-from-the-existing-backup) for the restore is out of date with this change.

Reply:
The last command worked after I did:

```bash
docker-compose -f local.yml stop django
```